smdebug/tensorflow/keras.py

```diff
@@ -736,6 +741,46 @@ def _prepare_collections_for_tf2(self):
             self._unwrap_model_with_input_output_saver()
         self.prepared_tf2_collections = True
 
+    def _handle_start_step_python_profiling(self, mode, current_step):
+        """
+        This function is called to handle python profiling at the start of a step.
+        :param mode: ModeKeys.TRAIN ModeKeys.EVAL ModeKeys.PREDICT
+        """
+        if self.python_profiler:
+            self.python_profiler.stop_profiling(
```

Comment:
Handle the if-case inside of `stop_profiling`

Reply:
Done